### PR TITLE
UWC: Added a Sample Publisher.

### PR DESCRIPTION
**Pull request by aditisa1**
Added a database sample publisher for data persist.

Signed-off-by: aditi saxena <aditi.saxena@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +143,34 @@ Used to uninstall & remove the UWC installation.
     The default retention period is set to 24 hours. Although the field "retention" in config.json is set to 23 hours, it includes a default shard duration of 1 hour which totally accounts to 24 hours of data retention.
     Refer influxDB documentation "https://www.influxdata.com/blog/influxdb-shards-retention-policies/" for more details.
 
+
+## Sample DB Publisher
+
+A sample database publisher which writes sample data into the influx database without getting involved with any of the UWC services.
```
Comment (nagdeepgk) [inline README.md:149]: Can be rephrased like this:
"A sample database publisher which publishes sample JSON data onto the EII MessageBus ZMQ broker. The JSON payload published on ZMQ broker is subscribed by the Telegraf & written to the influx database based on the "dataPersist" flag being true or false in input Json payload. (Writes to InfluxDB if "dataPersist" flag is true and doesn't write to DB if the flag is false).  This DB publisher app is containerized  & doesn't get involved with any of the UWC services."

File: README.md [hunk 2]
```diff
@@ -142,6 +143,34 @@ Used to uninstall & remove the UWC installation.
...
+
+Following are the steps for running it.
+
+1. Make sure [EmbPublisher APP](../tools/EmbPublisher/) is present after the repo is cloned.
+
+2. The UWC Embpublisher config and json file is present [eii_configs](./eii_configs/service_specific_cfgs/EmbPublisher/) .
```
Comment (nagdeepgk) [inline README.md:155]: Rephrase :
"The UWC Embpublisher config and json file" -> "The UWC Embpublisher's config.json and the json input file is present in ..".

File: build_scripts/01_uwc_pre_requisites.sh [hunk 3]
```diff
@@ -127,7 +127,12 @@ modify_config()
    			"Telegraf")
 			   cp $service_search_dir/Telegraf/* ${working_dir}/../../Telegraf/config/Telegraf/
 			   echo "Overridden the Telegraf service specific config files"
+			   break
    			;;
+		    "EmbPublisher")
+               cp $service_search_dir/EmbPublisher/UWC_Sample_DB_Publisher_config.json ${working_dir}/../../tools/EmbPublisher/datafiles/
```
Comment (nagdeepgk) [inline build_scripts/01_uwc_pre_requisites.sh:133]: Looks like a formatting issue.

File: eii_configs/service_specific_cfgs/EmbPublisher/config.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (nagdeepgk) [inline eii_configs/service_specific_cfgs/EmbPublisher/config.json:1]: I think this is config.json which we cn keep it directly inside ei_configs renaming like other services.

File: build_scripts/01_uwc_pre_requisites.sh [hunk 5]
```diff
@@ -127,7 +127,12 @@ modify_config()
    			"Telegraf")
 			   cp $service_search_dir/Telegraf/* ${working_dir}/../../Telegraf/config/Telegraf/
 			   echo "Overridden the Telegraf service specific config files"
+			   break
    			;;
+		        "EmbPublisher")
+                           cp $service_search_dir/EmbPublisher/UWC_Sample_DB_Publisher_config.json ${working_dir}/../../tools/EmbPublisher/datafiles/
+			   cp $service_search_dir/EmbPublisher/config.json ${working_dir}/../../tools/EmbPublisher/
```
Comment (aditisa1) [inline build_scripts/01_uwc_pre_requisites.sh:134]: this looks okay in full view
https://github.com/open-edge-insights/uwc/blob/5de961892d046d4d4a3d1524bb66fb42445786c1/build_scripts/01_uwc_pre_requisites.sh

**Review by nagdeepgk — APPROVED** [review]

**Pull request merged**